pkg/vadmin/remove_sc_vc.go

```diff
@@ -70,6 +70,8 @@ func (v *VClusterOps) genRemoveSubclusterOptions(s *removesc.Parms, certs *HTTPS
 	opts.DataPrefix = v.VDB.Spec.Local.DataPath
 	opts.IsEon = v.VDB.IsEON()
 	opts.ForceDelete = true
+	opts.PrimaryUpHost = s.InitiatorIP
```

Comment:
Why weren't we using this before?

Reply:
This is a new one I added to vclusterOps two weeks ago. Before, remove_subcluster doesn't do re_ip. Now I use this one and NodeNameAddressMap to do re_ip in remove_subcluster.